lms/djangoapps/learner_recommendations/utils.py

```diff
@@ -209,3 +209,29 @@ def get_cross_product_recommendations(course_key):
     Helper method to get associated course keys based on the key passed
     """
     return settings.CROSS_PRODUCT_RECOMMENDATIONS_KEYS.get(course_key)
+
+
+def get_active_course_run(course):
+    """
+    Returns an active course run based on prospectus frontend logic
+    for what defines an active course run
+    """
+    reviewed_status = "reviewed"
+    course_runs = course.get("course_runs")
+    course_run_statuses = course.get("course_run_statuses")
+    advertised_course_run_uuid = course.get("advertised_course_run_uuid")
+
+    if (
+        course_run_statuses
+        and len(course_run_statuses) == 1
+        and course_run_statuses[0].lower() == reviewed_status
```

Comment:
What does this code do? Why are we checking that the length of course_run_statuses is one and that it must be equal to "reviewed"?

Reply:
Because we got rid of the `marketable_course_runs_only` query string when querying discovery, we would be returning courses that may have many course_runs, and we may not know the statuses of those course_runs. Although the current courses that we are querying will always have a status of `published`, this logic was added here in the odd event that, for some reason, we weren't working with a course that has a published course_run; it is essentially the same as the Prospectus logic for defining an active course run in case the course it's working with does not have an `advertised_course_run_uuid` value. 

I'll be removing though, as I see how it's essentially unnecessary logic that will never be hit in the first place since the courses that have been chosen for our experiment will always have a published course run and we can just rely of the `advertised_course_run_uuid` to tell us which course run is the active one; this was just added for caution reasons when thinking about how removing the query string would affect the course runs we were working with